pkg/api/routes/inventory.go

```diff
@@ -9,10 +9,12 @@ import (
 	"go.uber.org/zap"
 )
 
-func processInband(c *fleetdb.Client, srvID uuid.UUID, dev *common.Device, log *zap.Logger) error {
+func processInband(c *fleetdb.Client, srvID uuid.UUID, dev *common.Device, log *zap.Logger) error { //nolint
```

Comment:
Because I didn't implement this function and the linter will complain about unused fleetdb client. 
Will remove the `nolint` once we have everything.